ppl/cmd/zar/README.md

```diff
@@ -265,54 +278,54 @@ zar ls custom.zng
 ```
 To see what's in it:
 ```
-find $ZAR_ROOT -name custom.zng | head -n 1 | xargs zq -f table 'head 10' -
+find $ZAR_ROOT -name idx-`for file in $ZAR_ROOT/indexdefs/*; do   zq -z $file | grep 'name:"custom.zng"' > /dev/null; if [ $? = 0 ]; then echo $file | sed 's/.*indexdefs\/idxdef-//'; fi done` | head -n 1 | xargs zq -f table 'head 10' -
```

Comment:
Why yes... that `for` loop in the backticks _is_ ugly! :smile:   The problem here is similar in spirit to the one described in #2412, in that the opaque naming of the index files left me scratching my head wondering how to best guide the user toward looking at the contents of one of them going off the index name. Since this is a "show me" moment and not something we expect users would do during production use, I'm ok leaving this as-is, but I'm open to counterarguments.

Reply:
```suggestion
find $ZAR_ROOT -name idx-$(zar index ls -f zng | zq -f text 'desc="zql-custom.zng" | cut id' -).zng | head -n 1 | xargs zq -f table 'head 10' -
```

Reply:
This doc change is good for now but this is an area where we will improve the UX with a better model for naming things inside of a lake... space, subspace, meta-layer, etc...